tests/blacken_docs_test.py

```diff
@@ -345,3 +345,47 @@ def test_integration_ignored_syntax_error(tmpdir, capsys):
         'f(\n'
         '```\n'
     )
+
+def test_format_src_rst_jupyter_sphinx():
```

Comment:
hehe looks like you didn't run the pre-commit hooks

Reply:
Yes, when I run them it changes A LOT of code, also a lot that I didn't touch.

I am not sure what I am doing wrong...

Reply:
huh!  can you show some diffs, that's kind of surprising!

Reply:
Hmmm, I cannot reproduce.

When I ran `pre-commit run --all` before, it also ran `black`.

Something must have been in an incorrect state.